src/status_im/multiaccounts/recover/core.cljs

```diff
@@ -160,15 +158,21 @@
    (navigation/navigate-to-cofx :recover-multiaccount-enter-phrase nil)))
 
 (fx/defn proceed-to-import-mnemonic
-  {:events [:multiaccounts.recover/enter-phrase-next-pressed]}
-  [{:keys [db] :as cofx}]
+  {:events [:multiaccounts.recover/phrase-validated]}
+  [{:keys [db] :as cofx} phrase-warnings]
   (let [{:keys [password passphrase]} (:intro-wizard db)]
-    (if (check-phrase-warnings passphrase)
+    (if-not (= (get-in (types/json->clj phrase-warnings) [:error]) "")
```

Comment:
what if error nil ? and you can use just get here, `(if (string/blank? (:error (types/json->clj phrase-warnings)) import show-error)

Reply:
Made an update, though not sure about that `import show-error` since I thought we were using the `popover` modal to display the error here.  Otherwise would be a UI change I think